### ISSUE TITLE
validation loss doesn't decrease

### ISSUE THREAD

**Issue by maxim28** [happypepper/DeepHoldem#25]
hi, happypepper,

i have generated 120w data for river street and try to train the network, but after 100 epochs i found the validation loss comes to 0.052-0.053 and never decrease any more. i have tried to lower the learning rate more after 100 epochs but nothing changed( the original code divide the lr by 10 after 50 epochs).

So, could you tell you how can you train the river network to decrease validation loss to 0.0415 for 100w data? is there any tricks? 

thanks a lot~

**Comment by aikupoker** [comment]
Hi @maxim28 

You can read a little more about how can improve more your DeepStack in this README.md file. There is a table with different data that you can compare:

https://github.com/aikupoker/deeper-stacker#performance
